website/vercel.json

```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/docs/verified-adapters",
```

Comment:
should we have any concerns about the source being verified adapters which we removed?

Reply:
good question, @amychen1776 ! would redirecting to this page https://docs.getdbt.com/docs/supported-data-platforms be better?

Reply:
per zoom chat, will go with the Supported data platforms page